src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -257,8 +257,8 @@ public String getColumnValue(final VariantContext variant) {
     AS_SB_TABLE { // Required
         public String getColumnValue(final VariantContext variant) {
             String out = getAttribute(variant, GATKVCFConstants.AS_SB_TABLE_KEY, null);
-            String outNotAlleleSpecific = variant.getGenotype(0).getExtendedAttribute(GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY, null).toString();
             if (out == null) {
```

Comment:
what does this switch do?

Reply:
oh---was this a different bug?

Reply:
this was more that we were trying to parse out the "fallback" (non-AS) annotation even if we didn't need it.  Now it just parses it if we can't find the first one.  The tests caught it